packages/job-worker/src/ingest/rundownInput.ts

```diff
@@ -432,6 +437,36 @@ export async function handleUpdatedSegmentRanks(
 	)
 }
 
+function updateAdLibActionRanks(segmentExternalId: string, newRank: number, cache: CacheForIngest): void {
```

Comment:
It would be nice for clarification that `newRank` is the `newSegmentRank`.

Reply:
I'm on the fence, but I have updated it.